src/core/types.d.ts

```diff
@@ -20,6 +20,11 @@ type FeeRates = {
   withdrawFeeFixed: number
 }
 
+type WithdrawFeeRange = {
```

Comment:
Don't add any code to `src/core` we are deleting all that code. It should be defined close to the `checkedToXxx` is defined.

Reply:
Fixed